airbyte-webapp/src/components/EntityTable/components/StreamStatusIcons/Checkmark.tsx

```diff
@@ -0,0 +1,9 @@
+export const Checkmark = () => (
```

Comment:
Should these icons go in `src/components/icons`? Is there anything special that makes them unique to this component?

Reply:
Nope, I'll move.